gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -1246,23 +1248,27 @@ function plymeta:GetHeightVector()
     end
 
     if matrix then
-        local pos = matrix:GetTranslation()
+        pos = matrix:GetTranslation()
+    end
 
+    -- if a player is too far away, their location is only updated sporadically
+    -- if the distance between these two positions is too large, the fallback position
+    -- should be used
+    if (pos - posPlayer):Length() < 90 then
```

Comment:
pos might be `nil` here

moving this inside the if-clause above should fix this imo. do you agree?

Reply:
That's absolutely true, thanks. I fixed it a bit differently as I think that way is the code more readable